### DISCUSSION TITLE
Feature: Real-time ETL

### DISCUSSION THREAD

**Discussion by ChenpiDog** [databendlabs/databend#8496] [Ideas]
Real-time Data Warehouse and real-time ETL can be realized only with SQL. 
![image](https://user-images.githubusercontent.com/55483982/197734932-5fdc6792-273b-4fe0-8bad-8a6598c3d9d7.png)

**Comment by cdmikechen** [comment]
This seems like a great idea, but I'd actually like to focus more on how the databend allows the next level to sense changes in the data on the upper level.

**Reply by ChenpiDog** [reply]
A mechanism like Change Data Capture (CDC)? It may involve technologies such as database logs and materialized views.

**Reply by cdmikechen** [reply]
@ChenpiDog 
My personal understanding of parquet is that it is a columnar storage format and probably won't handle CDC as well as rdbms.

**Reply by bohutang** [reply]
@cdmikechen 
We will handle the CDC in fuse engine(like the `Merge` statement), now it supports insert/select/delete, and the next plan is to support updates, all in one.
